### ISSUE TITLE
Expected response code 250 but got code "535", with message "535-5.7.8 Username and Password not accepted. Learn more at 535 5.7.8  https://support.google.com/mail/?p=BadCredentials x8-v6sm5005123itf.1 - gsmtp " {"exception":"[object] (Swift_TransportException(code: 535): Expected response code 250 but got code \"535\", with message \"535-5.7.8 Username and Password not accepted. Learn more at 535 5.7.8  https://support.google.com/mail/?p=BadCredentials x8-v6sm5005123itf.1 - gsmtp

### ISSUE THREAD

**Issue by sandystorm1990** [swiftmailer/swiftmailer#1083]
| Q                   | A
| ------------------- | -----
| Bug report?         | yes/no
| Feature request?    | yes/no
| RFC?                | yes/no
| How used?           | Standalone/Symfony/3party
| Swiftmailer version | x.y.z
| PHP version         | 7.2.1

### Example
Expected response code 250 but got code "535", with message "535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials x8-v6sm5005123itf.1 - gsmtp
" {"exception":"[object] (Swift_TransportException(code: 535): Expected response code 250 but got code \"535\", with message \"535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials x8-v6sm5005123itf.1 - gsmtp

**Comment by sandystorm1990** [comment]
how to solve this problem?

**Comment by tomzx** [comment]
Provide the appropriate username/password credentials?

**Comment by AltayAkkus** [comment]
Hello @sandystorm1990,
firstly: Welcome! It's my first time too.

## As an tip:
Do you see the table in your post? Filling out the necessary forms is increasing your chances to get help enormously. 

Same thing for the Observed behaviour and Expected behaviour fields.

And for the error message you included, highlighting it with a `code` tag (in the normal editor the little <> symbol) makes it far more readable. 

## To your problem:
Most Error Messages are pretty straightforward, so maybe formatting it would help you to comprehend what happened:
```
Expected response code 250 but got code "535", 
with message
 "535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8 https://support.google.com/mail/?p=BadCredentials x8-v6sm5005123itf.1 - gsmtp
" {"exception":"[object] (Swift_TransportException(code: 535): Expected response code 250 but got code "535", with message "535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8 https://support.google.com/mail/?p=BadCredentials x8-v6sm5005123itf.1 - gsmtp
```
So basically it says that it expected the response code to be [250](http://www.supermailer.de/smtp_reply_codes.htm) (the code means OK, everything worked) but instead of an OK he got 534, that means `SMTP Authentication unsuccessful/Bad username or password`

So basically the server is responding: Sorry, I couldn't log in, the credentials are false.

Could you please double check your credentials, mistakes happen.

If you double checked your credentials, and the problem still occurs, please respond to this message.

Have fun coding!

**Comment by OdaiTu** [comment]
did you solve the problem ?

**Comment by Raushn** [comment]
how to solve this problem?in laravel 5.6 when using smtp to send email
Error:  

Swift_TransportException (535)
Expected response code 250 but got code "535", with message "535-5.7.8 Username and Password not accepted. Learn more at 535 5.7.8 https://support.google.com/mail/?p=BadCredentials c19-v6sm3904798pfn.182 - gsmtp "

**Comment by fabpot** [comment]
Closing as Github is only used for bugs and feature requests. You should use StackOverflow for support instead. Thanks for your understanding.

**Comment by RMPR** [comment]
https://stackoverflow.com/questions/37290204/cannot-send-email-from-yii-swiftmail-to-gmail-expected-response-code-250-but-go/37290443

Excuse me to write here, but this thread was pretty useful to me and I thought it might be useful in this case + it's a redirection to stackoverflow ;)

**Comment by Developer-MR** [comment]
do not use # symbol on password section.. it work for me.

**Comment by shnslmz** [comment]
Hi guys,

It is related to Gmail side, this is solution:


1. enable the 2-step verification to google [HERE](https://www.google.com/landing/2step/)
2. Create App Password to be use by your system [HERE](https://security.google.com/settings/security/apppasswords)
3. I selected Others (custom name) and clicked generate
4. than, update your creds.

[Source](https://stackoverflow.com/questions/42558903/expected-response-code-250-but-got-code-535-with-message-535-5-7-8-username/50251189#50251189)
